tests/unit/Collectors/JobCollectorTest.php

```diff
@@ -104,6 +104,7 @@ public function testJobProcessingListenerIgnored()
     {
         $this->patternConfigReturn(self::JOB_IGNORE_PATTERN);
         $this->jobMock->shouldReceive('resolveName')->once()->andReturn(self::JOB_NAME);
+        $this->jobMock->shouldReceive('payload');
```

Comment:
@countless-integers  I added this to make the test pass, I guess L11 introduced a new internal method? The thing is, if I do `$this->jobMock->shouldReceive('payload')->once();` locally it fails:

```
  Method payload(<Any Arguments>) from Mockery_0_Illuminate_Contracts_Queue_Job should be called
   exactly 1 times but called 0 times.
```

Not sure if I'm doing the right thing 🤷🏻‍♂️

Reply:
[The method exists in L10 too](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Contracts/Queue/Job.php#L26). Not sure if I'm reading the test right, but it seems like it's assuming that the job would not be processed and that method would not be called.

Reply:
[I don't see anything in the upgrade guide on Job or Worker changes](https://laravel.com/docs/11.x/upgrade#queues). All the other tests use the same pattern, the only difference is how this is implemented for processing -- in others get name is called first, but in processing timer is started first. 

But if it passes if you add the mock then it could be that there's something else trying to process the job first. Not sure why would that happen in a unit test though.

Reply:
>   Method payload(<Any Arguments>) from Mockery_0_Illuminate_Contracts_Queue_Job should be called
>   exactly 1 times but called 0 times.

Any chance you're not testing with Laravel 11 locally?

Reply:
> [The method exists in L10 too](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Contracts/Queue/Job.php#L26). Not sure if I'm reading the test right, but it seems like it's checking if the job would be ignored based on the name pattern and the fact that `payload` is called, to me, indicates that it's not.

This is the reason the test is failing, it was added on L11.

https://github.com/laravel/framework/blob/11.x/src/Illuminate/Log/Context/ContextServiceProvider.php#L39